packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -26,23 +24,22 @@ module.exports = function (defaults) {
       ],
     }
   });
-
   return app.toTree();
 };
 `,
     tests: {
-      unit: getTestFiles(
+      unit: getTestFiles([
         'with-hooks-test.js',
         'without-hooks-test.js',
-        'with-multiple-modules-test.js'
-      ),
+        'with-multiple-modules-test.js',
+      ]),
     },
   });
 }
 
 async function classicInRepoAddon(project) {
   await classic(project);
-  await addInRepoAddon(project, 'fake-addon');
+  await addInRepoAddon(project, ['fake-addon', 'fake-addon-two']);
```

Comment:
Rather than making this take multiple names, I would probably have called this twice in a row. What happens if we want different version numbers for each addon?

Reply:
I separated this out to another scenario test and calling addInRepoAddon twice.